src/frontend/@types/mapeo-schema.d.ts

```diff
@@ -0,0 +1,143 @@
+// Type definitions for mapeo-schema 3.0.0
```

Comment:
Ideally we'd update `mapeo-schema` and pull that into this PR, but I think given the upcoming work that'll affect the schema, this definition file may be a more reasonable option in the short-term